apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/EvaluationEditor/Editor/Playground/index.tsx

```diff
@@ -19,6 +31,135 @@ import { Header } from '../Header'
 import Chat, { EVALUATION_PARAMETERS } from './Chat'
 import Preview from './Preview'
 
+const PARAMETERS = [
+  'messages',
+  'context',
+  'response',
+  'prompt',
+  'parameters',
+  'config',
+  'duration',
+  'cost',
+]
+
+const Variables = ({ providerLog }: { providerLog?: ProviderLogDto }) => {
+  if (!providerLog) return null
+
+  const collapsedContent = (
+    <div className='flex gap-2'>
+      {PARAMETERS.map((param) => (
+        <Badge variant='accent'>{param}</Badge>
+      ))}
+    </div>
+  )
+
+  const expandedContent = (
+    <div className='flex flex-col gap-4'>
+      <div className='flex flex-col gap-2'>
+        <div>
+          <Badge variant='accent'>messages</Badge>
+        </div>
+        <div className='rounded-lg border bg-secondary p-4 overflow-y-auto max-h-[300px]'>
+          <div className='flex flex-col gap-2'>
+            {providerLog.messages.map((message, index) => (
+              <div key={index} className='flex flex-col gap-1'>
+                <div>
+                  <Badge variant={roleVariant(message.role)}>
+                    {capitalize(message.role)}
+                  </Badge>
+                </div>
+                <div className='pl-4'>
+                  <Text.H6M whiteSpace='preWrap'>
+                    {printMessageContent(message.content)}
+                  </Text.H6M>
+                </div>
+              </div>
+            ))}
+            {/* Add the assistant's response */}
+            <div className='flex flex-col gap-1'>
+              <div>
+                <Badge variant={roleVariant('assistant')}>Assistant</Badge>
+              </div>
+              <div className='pl-4'>
+                <Text.H6M whiteSpace='preWrap'>
+                  {printMessageContent(providerLog.response)}
+                </Text.H6M>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div>
+        <Badge variant='accent'>context</Badge>
+        <textarea
+          className='w-full h-[150px] mt-2 p-2 rounded-lg bg-secondary resize-y text-xs'
```

Comment:
Let's avoid random sizes. tailwind has harmonic scales that are almost the same
<img width="722" alt="image" src="https://github.com/user-attachments/assets/7c4beb5a-398b-473a-a1d2-e2ce82015b0d">